README.md

```diff
@@ -88,7 +88,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-20.04, windows-2019, macOS-11]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
```

Comment:
```suggestion
        os: [ubuntu-latest, windows-latest, macOS-13]
```

Reply:
Should this have 14 too?

Reply:
The change to add macos-14 is in #1700. I wasn't sure why macos-13 never made it to `-latest`, I figured it might not be as battle-tested as macos-12. Also since we run our unit tests on macos-12 that seems like the best option to recommend to me.

Reply:
happy to go to macos-13 if we update our tests to that too though.